db/migrate/20170729143508_add_readnewsitems_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddReadnewsitemsToUser < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :readnewsitems, :string
+  end
+end
```

Comment:
Hela den här grejen skulle jag kanske modellerat som en ```has_many :through```-association:
http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association

Men om du gärna vill ha den direkt på User-modellen är ett automatiskt serialiserat objekt (en array) ett bättre alternativ än att bygga en egen implementation:
http://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/Serialization/ClassMethods.html

Alltså bara
```ruby
class User
  serialize :read_news_items, Array
end

class NewsItem
  def read_by_user?(user)
    user.read_news_items.include?(id)
  end

  def read_by_user!(user)
    user.read_news_items << id
    user.save!
  end
end
```